x-pack/osquerybeat/Jenkinsfile.yml

```diff
@@ -0,0 +1,67 @@
+when:
```

Comment:
Is it required ARM support? If so, the ARM stage is missing similar to the packaging stage.

Reply:
This currently uses the official distro of osquery, and at the time this project started there were no ARM official distro. The linux arm distro is now available in beta https://osquery.io/blog/linux-arm64-beta-support, could add it later.

Another and possibly better option for GA is to build osquery for all the platforms from source and host in our artifactory. This would allow us to avoid packaging official distros  .msi or .pkg for windows and mac into the distribution and build any flavor we need possibly earlier:
https://osquery.readthedocs.io/en/latest/development/building/

Anybody can assist with settings this up on our CI infrastructure?

Reply:
> Anybody can assist with settings this up on our CI infrastructure?

If the signing is required, the unified release process supports such a service, though we need to run the build in another CI. But if required, we can start a conversation with the team unified release team about where to host those binaries.

Reply:
@aleksmaus I think we can skip arm64 for 7.13 since it's only in beta on osquery and add it later when it's out of beta.

Reply:
yeah, that's the plan: to land what we have and then iterate.